mods/common/chrome/settings-display.yaml

```diff
@@ -192,15 +168,15 @@ Container@DISPLAY_PANEL:
 					Width: PARENT_RIGHT - 24
 					Height: 20
 					Children:
-						Container@PAUSE_SHELLMAP_CHECKBOX_CONTAINER:
+						Container@CURSORDOUBLE_CHECKBOX_CONTAINER:
 							X: 10
-							Width: PARENT_RIGHT / 2 - 10
+							Width: PARENT_RIGHT / 2 - 20
 							Children:
-								Checkbox@PAUSE_SHELLMAP_CHECKBOX:
+								Checkbox@CURSORDOUBLE_CHECKBOX:
 									Width: PARENT_RIGHT
 									Height: 20
 									Font: Regular
-									Text: Pause Menu Background
+									Text: Increase Cursor Size
```

Comment:
Maybe move this to the new accessibility section?

Reply:
sure

Reply:
maybe in follow up PR's?

Reply:
Okay